test/models/purpose_status_test.rb

```diff
@@ -3,6 +3,7 @@
 module Katello
   module PurposeStatusTests
     def test_status_matched
+      Organization.any_instance.stubs(:simple_content_access?).returns(false)
```

Comment:
I see you've added this stub to each test - while it works it seems like we can take some measure to DRY this up. If you add a setup method to the PurposeStatusTest class in this file, you can achieve that:

```
    def setup
      host.organization.stubs(:simple_content_access?).returns(false)
    end
```

Notice how this stubs `host.organization` vs `Organization.any_instance`. This is the preferred way to do it whereever possible so that we stub the minimum amount of behavior

Reply:
Updated, I see how in the area I wrote my main test in how there was a setup method. 

Looking closer at how it works I can see that it creates the top level setup for the test, that the methods can either use ie org, or a setting that will apply to all test methods. 

Let me know if I am reading that correct.

Reply:
Yep - the setup method is run before each test! https://www.rubypigeon.com/posts/minitest-cheat-sheet/